torch/testing/_internal/opinfo/definitions/_masked.py

```diff
@@ -769,7 +769,7 @@ def sample_inputs_masked_normalize(op_info, device, dtype, requires_grad, **kwar
         supports_forward_ad=True,
         supports_fwgrad_bwgrad=True,
         promotes_int_to_float=True,
-        dtypes=all_types_and_complex_and(torch.float16, torch.bfloat16, torch.bool),
+        dtypes=floating_and_complex_types_and(torch.float16, torch.bfloat16),
```

Comment:
Hm I think you can update the skips below and keep torch.bool in here. That way you can exercise the ValueErrors above.

Reply:
Okay, I've added an xfail for any calls using bool (I also removed some other skips that I think are redundant once bool is xfailed across for all calls)